packages/material-ui/src/Typography/Typography.test.js

```diff
@@ -1,7 +1,14 @@
-// @ts-check
```

Comment:
Unfortunately the new variants API requires augmenting the ts module, but this is not a ts file... Not sure what's would be the best approach here.. We may add these tests as a separate file and leave the `// @ts-check` here

Reply:
Or we could add `@ts-ignore` where necessary